pulp_container/app/cache.py

```diff
@@ -37,6 +37,13 @@ def make_key(self, request):
         key = ":".join(all_keys[k] for k in self.keys)
         return key
 
+    async def make_response(self, key, base_key):
+        """Fix the Content-Type header to remove the charset."""
+        response = await super().make_response(key, base_key)
+        if response is not None:
+            response.headers["Content-Type"] = response.headers["Content-Type"].split(";")[0]
```

Comment:
is this case-sensitive?  it fails on getting `reponse.headers["Content-Type"]` from the original response